packages/http-client/package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@actions/http-client",
-      "version": "2.2.0",
+      "version": "2.2.1",
```

Comment:
🤔 was this just missed when http-client was updated to 2.2.1?

Reply:
👍🏾 it looks like it